src/core/localize.rkt

```diff
@@ -164,52 +177,84 @@
     (define exacts (list->vector (apply subexprs-fn pt)))
     (define actuals (apply actual-value-fn pt))
 
-    (for ([expr (in-list exprs-list)]
+    (for ([spec (in-vector spec-vec)]
+          [expr (in-list exprs-list)]
           [root (in-vector roots)]
           [exact (in-vector exacts)]
           [actual (in-vector actuals)]
           [expr-idx (in-naturals)])
-      (define err
+      (define true-err
+        (if true-err? ;; ??? Whats the default values for true error literal, variable approx and if?
```

Comment:
I'm not really thrilled about this being computed as part of `compute-local-error`. This code isn't needed during the Herbie core, just for an API method; so `compute-local-error` is now doing two tasks that aren't related.